src/sensors/bno055sensor.cpp

```diff
@@ -39,6 +39,11 @@ void BNO055Sensor::motionSetup() {
     imu.setAxisRemap(Adafruit_BNO055::REMAP_CONFIG_P0);
     imu.setAxisSign(Adafruit_BNO055::REMAP_SIGN_P0);
     m_Logger.info("Connected to BNO055 at address 0x%02x", addr);
+
+    //Adafruit BNO055's use external crystal. Enable it, otherwise it does not work.
+    delay(1000);
+    imu.setExtCrystalUse(true);
```

Comment:
Is there a case where the external crystal won't be present? For example, cheaper BNO055 breakout boards?

Reply:
No, this is the correct fix. The crystal was disasbled because first board I made didn't have it, but all others do. I thought we had this fix for a while though? or true was even default.

Reply:
`imu.setExtCrystalUse(false);` previously was in the code, and we removed it in https://github.com/SlimeVR/SlimeVR-Tracker-ESP/commit/658db609e893617bb41dd8ebed3ca6cf46af9391. Now I checked the datasheet and realized that we actually didn't fix anything, because internal oscillator is used by default, lmao.
![image](https://user-images.githubusercontent.com/6103913/197713045-ab2eb8ac-49c0-485c-9449-b69fbd6979aa.png)

Reply:
If there is an external crystal, it's probably better to use it, if all the breakout boards available use it then we could probably enable this

Reply:
I did check the manual, we should use `imu.setExtCrystalUse(true);` if the board does not have one, or if it is faulty, it will switch back to internal.

https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bno055-ds000.pdf
PDF Page 110
> 5.5.1 External 32kHz Crystal Oscillator
> When switching to the external clock source, the BNO055 will check for integrity of the crystal’s
> signal. In case of improper signal (e.g. crystal damaged, broken/missing...), the BNO055 will
> automatically switch back to its internal clock source and clear the CLK_SEL bit in
> SYS_TRIGGER register.
>
> It takes minimum ~600ms to configure the external crystal and startup the BNO055.

Reply:
Ah okay awesome, then this change is good 👍